timemachine/cpp/src/langevin_integrator.hpp

```diff
@@ -16,14 +16,15 @@ class LangevinIntegrator : public Integrator {
     const double dt_;
     const double friction_;
     double ca_;
+
+    // The offset into the current batch of noise
+    int noise_iteration_;
```

Comment:
nit: would `noise_offset_` be a more appropriate name?

Reply:
>  The explicitness seems useful, but there seems to be an inconsistency between this style and D being a parameter, since the code will be incorrect for D other than 3.

Should we consider adding a static assertion that `D` is 3 wherever we manually unroll making this assumption?